src/util/nft.js

```diff
@@ -11,6 +10,24 @@ import {
 let queryClient = null;
 let iscnLib = null;
 
+export const NFT_INDEXER_LIMIT_MAX = 100;
+
+export const ORDER_CREATED_CLASS_ID_BY = {
+  PRICE: 'PRICE',
+  ISCN_TIMESTAMP: 'ISCN_TIMESTAMP',
+};
+
+export const ORDER_COLLECTED_CLASS_ID_BY = {
+  PRICE: 'PRICE',
+  LAST_COLLECTED_NFT: 'LAST_COLLECTED_NFT',
+  NFT_OWNED_COUNT: 'NFT_OWNED_COUNT',
+};
```

Comment:
This can be merged as one object?

Reply:
Will it cause misusing in different scenarios?

Reply:
I think it is not intuitive for introducing `created` and `collected` concept for sorting

Reply:
Anyway, I think we keep separated for now

Reply:
OK